tests/integration/conftest.py

```diff
@@ -49,4 +49,12 @@ async def app(ops_test: OpsTest, app_name: str):
     await ops_test.model.add_relation(hello_kubecon_app_name, app_name)
     await ops_test.model.wait_for_idle(status=ActiveStatus.name, idle_period=120)
 
+    # Add ignore rule
```

Comment:
Should this be done for all tests? Or just a test specifically for the ignore rules?

Reply:
Changed all the integration tests, will push new version now.